community/modules/scheduler/schedmd-slurm-gcp-v5-controller/main.tf

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+locals {
+  ghpc_startup_script_controller = [{
+    filename = "ghpc_startup.sh"
+    content  = var.controller_startup_script
+  }]
+  ghpc_startup_script_compute = [{
+    filename = "ghpc_startup.sh"
+    content  = var.compute_startup_script
+  }]
+}
+
+module "slurm_controller_instance" {
+  source = "github.com/SchedMD/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_controller_instance?ref=v5.0.2"
+
+  access_config                = var.access_config
```

Comment:
I realize that there is some value in clustering alike configuration settings here (and below). But, at same time, I think it might make sense just to go fully alphabetical.

I am trying to look for duplicate settings in the template and in the instance and understand if duplication is a problem.